internal/namespaces/config/commands.go

```diff
@@ -29,10 +31,50 @@ func GetCommands() *core.Commands {
 }
 
 func configRoot() *core.Command {
+	configPath := scw.GetConfigPath()
+
 	return &core.Command{
-		Short:     `Config file management`,
-		Long:      `Manage your Scaleway CLI config file.`,
+		Short: `Config file management`,
+		Long: `Config management engine is common across all Scaleway developer tools (CLI, terraform, SDK,... ). It allows to handle your Scaleway config through two ways: environment variables or/and config file.
+
+Scaleway config file is self-documented, we recommend you to have a look at it at least once before using Scaleway developer tools: ` + terminal.Style(configPath, color.Bold, color.FgBlue) + `
+
+In this CLI, ` + terminal.Style(`config file attributes are applied prior to the environment variables`, color.Bold) + `: environment variable > config attribute.
+
+The following environment variables are supported:
+- ` + terminal.Style("SCW_ACCESS_KEY", color.Bold, color.FgBlue) + ` the access key of a token
+- ` + terminal.Style("SCW_SECRET_KEY", color.Bold, color.FgBlue) + ` the secret key of a token
+- ` + terminal.Style("SCW_DEFAULT_ORGANIZATION_ID", color.Bold, color.FgBlue) + ` your default organization ID
+- ` + terminal.Style("SCW_DEFAULT_REGION", color.Bold, color.FgBlue) + ` your default region
+- ` + terminal.Style("SCW_DEFAULT_ZONE", color.Bold, color.FgBlue) + ` your default availability zone
+- ` + terminal.Style("SCW_API_URL", color.Bold, color.FgBlue) + ` URL of the API
+- ` + terminal.Style("SCW_INSECURE", color.Bold, color.FgBlue) + ` set this to true to enable the insecure mode
+- ` + terminal.Style("SCW_PROFILE", color.Bold, color.FgBlue) + ` set the config profile to use
+
```

Comment:
Maybe we should specify that the access/secret keys and the organization id can be found in the [Scaleway console](https://console.scaleway.com/account/credentials)?